erpnext/stock/report/item_prices/item_prices.py

```diff
@@ -20,24 +21,47 @@ def execute(filters=None):
 	precision = get_currency_precision() or 2
 	data = []
 	for item in sorted(item_map):
-		data.append([item, item_map[item]["item_name"],
-			item_map[item]["description"], item_map[item]["stock_uom"],
+		data.append([item, item_map[item]["item_name"],item_map[item]["item_group"],
+			item_map[item]["description"],
 			flt(last_purchase_rate.get(item, 0), precision),
-			flt(val_rate_map.get(item, 0), precision),
+			sn.get(item, {}).get("supplier"),
+			sn.get(item, {}).get("supplier_name"),
+			sn.get(item, {}).get("supplier_part_no"),
```

Comment:
Supplier Part no is maintain in a table and hold multiple record based on supplier. What is the point to show a random supplier/supplier name / part no?